spring-cloud-streams/src/main/java/org/springframework/cloud/streams/config/ChannelBindingAdapterConfiguration.java

```diff
@@ -49,7 +49,7 @@
  *
  */
 @Configuration
-@ImportResource("classpath*:/META-INF/spring-xd/bus/codec.xml")
+@ImportResource("classpath*:/META-INF/spring-bus/codec.xml")
```

Comment:
I made `spring-bus -> spring-cloud-streams`. The XML should probably go away as well.